addon/src/modifiers/inputmask.js

```diff
@@ -54,7 +64,7 @@ export default class InputmaskModifier extends Modifier {
     );
   }
 
-  _setInputMask(args) {
-    new Inputmask(args).mask(this.element);
+  _setInputMask(element, args) {
```

Comment:
We should revert this back with just args parameter IMO.

Reply:
see previous comment